variables.tf

```
@@ -459,6 +459,12 @@ variable "metadata_http_tokens_required" {
   description = "Set true to require IMDS session tokens, disabling Instance Metadata Service Version 1."
 }
 
+variable "metadata_http_protocol_ipv6_enabled" {
+  type        = bool
+  default     = true
```

This should default to `false`, because the `launch_template` resource defaults this to `false`. (See: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_template.)

```suggestion
  default     = false
```